asapdiscovery-alchemy/asapdiscovery/alchemy/cli/alchemy.py

```diff
@@ -833,6 +833,14 @@ def stop(network_key: str):
     help="Make predictions using only the largest subnetwork present in the results. "
     "Useful in cases where the network is disconnected by e.g. simulation failures.",
 )
+@click.option(
+    "-wtop",
+    "--write-top-n-poses",
+    help="The number of top-scoring poses to write to a multi-SDF in the local directoy.",
```

Comment:
```suggestion
    help="The number of top-scoring poses to write to a multi-SDF in the local directory.",
```

Reply:
Also why not just write them all? There isn't much of a difference space wise unless im missing something.